filtering/myfilters.js

```diff
@@ -92,6 +92,13 @@ MyFilters.prototype._updateDefaultSubscriptions = function() {
         return;
     }
 
+    // Function that will add a new entry with updated id,
+    // and will remove old entry with outdated id.
+    function renameSubscription(old_id, new_id) {
+        that._subscriptions[new_id] = that._subscriptions[old_id];
+        delete that._subscriptions[old_id];
+    }
+
```

Comment:
@kpeckett This function has been declared over and over again, 'cause it was in a for loop below. So yeah, for performance reasons :)